src/modules/transaction/store/index.ts

```diff
@@ -106,4 +106,38 @@ export default class TransactionStore {
             this.rootStore.vigorStore.fetchGlobalStats(),
         ])
     }
+
+    @action borrow = async ({quantity}: { quantity: TAsset }) => {
+        if(!await this.checkLogin()) return;
+        await this.sendTransaction({
+            name: `assetout`,
```

Comment:
I think for the kind of transactions, we can set a constant file too

Reply:
For the name field? Hm, not sure if this would make it easier to read and maintain. I think it would just needlessly complicate things. Like this, the `borrow` function is self-contained and if something related to the borrow transaction changes, you just need to change things in this function here.

Or what was your idea?